client/state/initial-state.js

```diff
@@ -82,7 +83,13 @@ export function persistOnChange( reduxStore, serializeState = serialize ) {
 			.catch( ( setError ) => {
 				debug( 'failed to set redux-store state', setError );
 			} );
-	}, SERIALIZE_THROTTLE, { leading: false, trailing: true } ) );
+	}, SERIALIZE_THROTTLE, { leading: false, trailing: true } );
+
+	if ( global.window ) {
```

Comment:
Right so if you set a variable to the global scope eg `var foo = 'foo;'`, in node this will be accessible by `global.foo, while in a browser this would be `window.foo`;